app/src/main/java/com/asfoundation/wallet/referrals/ReferralInteractor.kt

```diff
@@ -59,4 +62,37 @@ class ReferralInteractor(
   private fun hasDifferentInformation(newInformation: String, savedInformation: String): Boolean {
     return newInformation != savedInformation
   }
+
+  override fun getReferralNotifications(): Maybe<List<ReferralNotification>> {
+    return getPendingBonusNotification().map { listOf(it) }
```

Comment:
why don’t you do the map on the chain? I don’t if there’s a need to create a second method